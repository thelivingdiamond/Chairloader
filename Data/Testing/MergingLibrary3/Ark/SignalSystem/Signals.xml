<?xml version="1.0"?>
<signals>
    <signal
        name="signal 1"
        id="111111"
        notes="" />
    <signal
        name="signal 2"
        id="222222"
        notes="" />
</signals>