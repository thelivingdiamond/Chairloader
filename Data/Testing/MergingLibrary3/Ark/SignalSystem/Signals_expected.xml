<?xml version="1.0"?>
<signals>
    <signal
        name="signal 1"
        id="111111"
        notes="" />
    <signal
        name="signal 2"
        id="222222"
        notes="" />
    <signal
        name="signal 3"
        id="333333"
        notes="Signal from mod" />
</signals>
